index.js

```diff
@@ -280,6 +291,19 @@ Plaid.getInstitutions = function(env, callback) {
   }, callback);
 };
 
+Plaid.searchInstitutions = function(options, env, callback) {
+  var qs = querystring.stringify(R.reject(R.equals(undefined), {
```

Comment:
Could we use `R.isNil` in place of `R.equals(undefined)`, or do we actually want to treat `null` and `undefined` differently?